zrml/prediction-markets/src/benchmarks.rs

```diff
@@ -397,47 +376,59 @@ benchmarks! {
         let d in 0..T::MaxDisputes::get();
 
         let c_u16 = c.saturated_into();
-        let (caller, market_id) = setup_resolve_common_categorical_after_dispute::<T>(a, b, c_u16)?;
+        let (caller, market_id) =
+            setup_resolve_common_categorical::<T>(a, b, c_u16)?;
+        T::MarketCommons::mutate_market(&market_id, |market| {
+            let admin = account("admin", 0, 0);
+            market.dispute_mechanism = MarketDisputeMechanism::Authorized(admin);
+            Ok(())
+        })?;
 
         for i in 0..c.min(d) {
```

Comment:
I think this should iterate over `0..d` and dispute ~~`OutcomeReport::Categorical(i % c)`.~~ `OutcomeReport::Categorical(i % 2)` to maximize the number of correct reporters, which will result in addition `deposit` calls in `on_resolution`.